src/Speckle.Sdk/Transports/TransportHelpers.cs

```diff
@@ -51,7 +51,8 @@ CancellationToken cancellationToken
         }
 
         targetTransport.SaveObject(kvp.Key, child);
-        sourceTransport.OnProgressAction?.Invoke($"{sourceTransport}", i++);
+        var count = i++;
+        sourceTransport.OnProgressAction?.Invoke(new ProgressArgs(ProgressEvent.UploadObject, count, closures.Count));
```

Comment:
We may need to look at this in the future. Sending can be performed to multiple transports at the same time, and previously we were reporting progress on each transport individually.

I'm not sure how this could be done here (or even if we'd want the same behaviour at all)

Reply:
I noticed on send that it does SQLite cache as well as sending.  Kind of behaves fine 😬